### PR TITLE
Do not allocate strings for TextWriter.Write

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/Serilog/Formatting/Json/JsonFormatter.cs [hunk 1]
```diff
@@ -399,25 +399,57 @@ static void WriteBoolean(bool value, TextWriter output)
 
     static void WriteSingle(float value, TextWriter output)
     {
+#if NET5_0_OR_GREATER
```
Comment (SimonCropp) [inline src/Serilog/Formatting/Json/JsonFormatter.cs:402]: can we re-use `FEATURE_SPAN` here?

Comment (sungam3r) [reply src/Serilog/Formatting/Json/JsonFormatter.cs]: done

**Comment by sungam3r** [comment]
Conflicts resolved.

**Review by SimonCropp — APPROVED** [review]

**Comment by SimonCropp** [comment]
@nblumhardt any objections?

**Review by nblumhardt — APPROVED** [review]

**Comment by SimonCropp** [comment]
cool. will merge once the build is green

**Pull request merged**